Runtime/SimpleGraphQL/HttpUtils.cs

```diff
@@ -441,5 +441,17 @@ await _webSocket.SendAsync(
                 break;
             }
         }
+        
+        public static string PrintDictionary<TKey, TValue>(this Dictionary<TKey, TValue> dict)
+        {
+            string print = string.Empty;
+            foreach (var pair in dict)
+            {
+                print += pair.Key.ToString() + " - " + pair.Value.ToString();
+                print += "\n";
+            }
+
+            return print;
+        }
```

Comment:
We'll use it quite a lot, the more efficient the better
```suggestion
        public static string PrintDictionary<TKey, TValue>(this Dictionary<TKey, TValue> dict)
        {
            StringBuilder print = new StringBuilder();
            foreach (var pair in dict)
            {
                print.AppendFormat("{0} - {1}\n", pair.Key,pair.Value);
            }

            return print.ToString();
        }
```